app/templates/components/ui-table/cell/events/view/sessions/cell-lock-session.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if record.isLocked}}
+  {{#ui-popup content=(t 'Unlock Session') class='ui icon button'  click=(action unlockSession record) position='left center'}}
```

Comment:
Make it a basic button like the other action buttons.

Reply:
@ritikamotwani all the action buttons are also `ui-popup` as we need to show tooltip on hovering the buttons displaying the specific task which they carry out

Reply:
Yes, I know what a popup is. I am talking about the class of the button used. Make it basic :)